proposals/json-td/index.html

```diff
@@ -71,37 +79,59 @@
       "type": "boolean,
       "description": "Whether the lamp is turned on",
       "href": "/things/lamp/properties/on"
+    },
+    "lightLevel" :
+    {
+      "type": "integer",
+      "description": "The level of light from 0-100",
+      "min" : 0,
+      "max" : 100,
+      "href": "/things/lamp/properties/lightLevel"
     }
   },
   "actions": {
     "toggle": {
+      "type" : "boolean",
```

Comment:
If a boolean value is provided then this action is just the same as setting the on property. I imagined this action toggling the lamp to on if it's off or off if it's on, which is what makes it an action rather than a property. Not a great example I admit, but this example was meant to mirror the example given in the W3C Thing Description spec https://w3c.github.io/wot-thing-description/#introduction